examples/slack_alert/run.py

```diff
@@ -52,15 +65,15 @@ def main(config: str):
             trainer=svc_trainer(),
             evaluator=evaluator(),
             formatter=test_acc_post_formatter(),
-            alerter=slack_alerter_post_step(),
+            alerter=slack_alerter_post_step(slack_token, slack_channel),
```

Comment:
Great idea, but this will require a few more changes. If you look at the definition of `slack_alerter_post_step`, it doesn't actually accept those two arguments yet:

```python
def slack_alerter_post_step(
    params: SlackAlerterParameters, context: StepContext, message: str
) -> bool:
```

To set the channel ID, we could pass it as part of the params argument:

```suggestion
            alerter=slack_alerter_post_step(params=SlackAlerterParameters(slack_channel_id=slack_channel)),
```

The token is a bit more challenging to parametrize though. Currently, it can only be set in the stack component configuration, i.e., you have to provide it when running `zenml alerter register ...`. We could also add it to the `SlackAlerterParameter` class, but then we will need to implement some logic for determining the correct token at run time, similar to `_get_channel_id` on line 72 of `src/zenml/integrations/slack/alerters/slack_alerter.py`.